.github/labeler.yml

```diff
@@ -0,0 +1,66 @@
+# Add 'awaiting-review' to any PR opened
+awaiting-review:
+- '**/*'
+
+'product: CLion':
+- clwb/**/*
+- cpp/**/*
+- base/**/*
+- examples/cpp/simple_project/**/*
+- intellij_platform_sdk/BUILD.clion213
+- intellij_platform_sdk/BUILD.clion221
+- intellij_platform_sdk/BUILD.clion222
+- intellij_platform_sdk/BUILD.clion223
+- intellij_platform_sdk/BUILD.clion231
+- intellij_platform_sdk/BUILD.clion232
+
+'product: Android Studio':
+- aswb/**/*
+- sdkcompat/**/*
+- examples/kotlin/**/*
```

Comment:
can we remove these 2? I think in most cases they will not affect ASwB

Reply:
So should this be part of product:intellij ? or remove from the categorising as it not part of any product label.

Reply:
you can move it to intellij

Reply:
Sure.Done.